concent_api/core/admin.py

```diff
@@ -1,44 +1,56 @@
 from django.contrib import admin
+from django.db.models import Q
 
-from common.admin    import ModelAdminReadOnlyMixin
-from .models        import PendingResponse
-from .models        import StoredMessage
-from .models        import Subtask
+from common.admin import ModelAdminReadOnlyMixin
+from common.helpers import get_current_utc_timestamp
+from common.helpers import parse_timestamp_to_utc_datetime
+from .models import PendingResponse
+from .models import StoredMessage
+from .models import Subtask
 
+ACTIVE_STATE_NAMES = [x.name for x in Subtask.ACTIVE_STATES]
+PASSIVE_STATE_NAMES = [x.name for x in Subtask.PASSIVE_STATES]
 
-class ActivePassiveStateFilter(admin.SimpleListFilter):
 
-    title          = 'Active/passive state'
+class ActivePassiveDownloadsStateFilter(admin.SimpleListFilter):
+    title = 'Active/passive/downloads state'
```

Comment:
There are a lot of style changes that only distract from real changes. Please put them all in a separate commit.

Reply:
Hmm I haven't seen any in this commit... I've made git diff with master and only one is this above but this change of styling is connected with change in variable value so I dont't think I need to do it in separate commit.

Reply:
There are more:
- Indentation change in imports above
- Removing a space before `'Active'`.

But OK. That's not that much. Let's leave it as is.